### PR TITLE
add link to APM info

### PR DESCRIPTION
**Pull request by swaretech**
seeing the link to related back end spans in APM is part of interpreting browser test results

**Requirements**

- [ x] Content follows Splunk guidelines for style and formatting.
- [ x] You are contributing original content.

**Describe the change**

Adding a line to what is provided in the waterfall. Using the link to related back end spans in APM is part of interpreting browser test results

### CODE REVIEW

File: synthetics/browser-test/browser-test-results.rst [hunk 1]
```diff
@@ -143,6 +143,7 @@ Using the waterfall chart, you can do the following:
 - Expand the details in a row to show the request and response headers for that resource.
 - Hover over a row of the timeline to view a pop-up message with detailed request timings for that resource.
 - Search resources in a page by keywords in the URL.
+- Follow a direct link to related back-end spans if the same app is :new-page:`instrumented with APM <https://docs.splunk.com/observability/en/synthetics/set-up-synthetics/set-up-synthetics.html#optional-link-synthetic-spans-to-apm-spans>`.
```
Comment (theletterf) [inline synthetics/browser-test/browser-test-results.rst:146]: We might want to use the ref here, not a static link. 

+CC @bhillmer

**Review by theletterf — CHANGES_REQUESTED** [review]

File: synthetics/browser-test/browser-test-results.rst (hunk 1 above)
Comment (bhillmer) [reply synthetics/browser-test/browser-test-results.rst:146]: @theletterf I agree. Not sure how to make changes to this branch though.

Comment (theletterf) [reply synthetics/browser-test/browser-test-results.rst:146]: @bhillmer Maintainers (us) are allowed to edit branches in forks as per repo settings:

Screenshot 2024-05-29 at 18 03 05

**Review by bhillmer — APPROVED** [review]

File: synthetics/browser-test/browser-test-results.rst (hunk 1 above)
Comment (bhillmer) [reply synthetics/browser-test/browser-test-results.rst]: Oh thank you! I never would have found that!

**Pull request merged**